src/tests/test_buildercore_terraform.py

```diff
@@ -87,13 +87,6 @@ def test_fastly_template_minimal(self):
                                 'ssl_check_cert': True,
                             }],
                             'default_ttl': 3600,
-                            'request_setting': [{
-                                'name': 'default',
-                                'default_host': 'prod--www.example.org',
-                                'force_ssl': True,
```

Comment:
the other settings like `force_ssl` are guaranteed anyway?

Reply:
Backend is port 443, so should always be HTTPS. Plus: https://github.com/elifesciences/builder/blob/9641e7d6109f6ad460377aac5c1717f4c7dec0a5/src/buildercore/terraform.py#L323-L327

Reply:
Oh hang on, forces the client's request to be HTTPS. Looks to no longer work.

Reply:
Restored (and improved) in 7e865f34b369fe1cad01c9e8ab46224512fffa8e.